shared/params/config.go

```diff
@@ -222,18 +222,17 @@ func MainnetConfig() *BeaconChainConfig {
 // DemoBeaconConfig retrieves the demo beacon chain config.
 // Notable changes from minimal config:
 //   - Max effective balance is 3.2 ETH
-//   - Ejection threshold is 3.175 ETH
-//   - Genesis threshold is disabled (minimum date to start the chain)
+//   - Ejection threshold is 1.6 ETH
 func DemoBeaconConfig() *BeaconChainConfig {
-	demoConfig := MinimalSpecConfig()
-	demoConfig.MinDepositAmount = 100
+	demoConfig := MainnetConfig()
+
+	demoConfig.MinDepositAmount = 0.1 * 1e9
 	demoConfig.MaxEffectiveBalance = 3.2 * 1e9
-	demoConfig.EjectionBalance = 3 * 1e9
+	demoConfig.EjectionBalance = 1.6 * 1e9
```

Comment:
why set it back to 1.6?

Reply:
Because this is mainnet configuration, except 1/10th of the deposits. 
Everything else feels mainnet, activation/ejection times, penalties, etc.

Reply:
Without proper incentive to reward validators participating well in testnet. Such low ejection balance will cause testnet to struggle and diverge from production environment

Reply:
I agree with you, which is why I am advocating for this change. Using the mainnet configuration threshold gives us the ability to test and detect this scenario. If it kills the testnet, then we get a great post-mortem.

Reply:
Sure. Although the scenario differs from production(because people will care to keep clients alive for as long as they can), but the end result is still the same. It's a good network fragmentation test